src/content/ModalLayer.tsx

```diff
@@ -94,6 +103,65 @@ function ModalLayer(props: ModalLayerProps) {
           hasTintedBg: false,
         }
       }
+      case ModalPreset.SyncErrors: {
+        const data: SyncScheduleModalData =
+          action.additionalData as SyncScheduleModalData
+
+        const errors = (
+          <ul style={{ fontSize: "1.2em", padding: "10px" }}>
+            {data.data.map((error, index) => (
+              <li key={index}>
+                {index + 1}. {error}
+                <br />
+              </li>
+            ))}
+          </ul>
+        )
+
+        return {
+          title: "Error Syncing With Saved Schedule",
+          body: errors,
+          closeButtonText: "Close",
+          actionButtonText: "Okay",
+          actionHandler: data.onConfirm,
+          cancelHandler: data.onCancel,
+          alignment: ModalAlignment.Center,
```

Comment:
These are nitpicks but:
- alignment is `Center` by default, so you can remove it here
- for the `actionButtonText`, I'd use "OK" instead of "Okay" (it just looks nicer, and modals usually use that lol)

Reply:
valid points, fixed!